frontend/src/screens/apps/Create.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+
+const Create = () => {
+  const pairingUri = "YOUR_PAIRING_URI"; // Replace with actual data or props
+  const [copied, setCopied] = useState(false);
+
+  // useEffect(() => {
+  //   // This would replace the script to create QR Code
+  //   QrCreator.render(
+  //     {
+  //       fill: window.matchMedia('(prefers-color-scheme: dark)').matches ? "#FFF" : "#000",
+  //       text: pairingUri,
+  //       size: 250,
+  //     },
+  //     document.getElementById("connect-qrcode")
+  //   );
+  // }, [pairingUri]);
+
+  const copyToClipboard = async () => {
+    if (navigator.clipboard && window.isSecureContext) {
+      await navigator.clipboard.writeText(pairingUri);
+    } else {
+      // Fallback for older browsers
```

Comment:
this should be updated to React. Or we can use a copy to clipboard library that supports old browsers

Reply:
This is how most are doing even in React, shouldn't be an issue